runtime/vm/JFRConstantPoolTypes.hpp

```diff
@@ -1246,6 +1268,67 @@ class VM_JFRConstantPoolTypes {
 		gcConfiguration->heapAddressBits = J9JAVAVM_REFERENCE_SIZE(vm) * 8;
 	}
 
+	uintptr_t processNativeLibrariesCallback(const char *libraryName, void *lowAddress, void *highAddress, void *userData)
```

Comment:
you should add a `_nativeLibraryCMDlineStrSizeTotal` and ` += strlen(libraryName)` to track the size of the strings, that way you can get a more accurate `requiredBufferSize`